sync-bucket-pipeline.yml

```diff
@@ -0,0 +1,30 @@
+---
+jobs:
+- name: sync-bucket
```

Comment:
At some point we should do `sync-((job-0-name))-bucket` so it's much easier to template and have as one pipeline that does syncing, but this works for now.

Reply:
I don't like the idea of monolithic pipelines that span concerns. In my opinion, pipelines should strive to be the lifecycle of a thing, not a grouping of all sync functions.

Reply:
I think I'm with Steve on this one